pkg/web/templates/sunrise_404.html

```diff
@@ -0,0 +1,15 @@
+{{ template "sunrise" . }}
+{{ define "content" }}
+
+<section class="text-center py-12">
+  <img src="/static/404error.svg" alt="404 Error" class="w-[350px] mx-auto" />
+  <p class="py-4 text-xl">Sorry, we couldn't find the page you're looking for. Try going back to a previous page or contact <a href="mailto:support@rotational.io" class="text-blue-700">support@rotational.io </a>for assistance.</p>
+  <p class="py-4 text-xl">If you're not yet part of the TRISA network, consider joining to streamline future Travel Rule compliance and secure data exchanges.</p>
+
+  <div class="my-10 flex items-center gap-x-4 max-w-sm mx-auto">
+    <a onclick="window.history.back()" class="flex-1 p-4 rounded-md bg-black text-white font-semibold hover:bg-black/80">Go Back</a>
```

Comment:
I thought that adding a back button could be helpful, but if a user doesn't have a page to go back to we don't currently have a fallback option (ex. login page) so this can be removed.

Reply:
It's a good thought - I think the only reason a user would get to this page is if they clicked on a link in an email; so probably they wouldn't be able to make too much use of a back button.

What about a "learn more" link/button instead? E.g. provide two choices: learn more or join TRISA network

Reply:
I was thinking about that. I'll add a learn more link.